tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37,pypy,
+    py27,py35,py36,py37,py38,pypy,
     py27-pyramid{13,14,15,16,17,18,19},
-    py27-pyramid{13,14,15,16,17,18,110},
```

Comment:
Pyramid 1.10 still supports Python 2.7

Reply:
That's fine, but the line above it also specifies py27... it was defined twice for no good reason.